code/src/java/pcgen/io/exporttoken/WeaponToken.java

```diff
@@ -1550,7 +1548,7 @@ else if (display.isSecondaryWeapon(eq) && display.hasPrimaryWeapons())
 			else
 			{
 				// offhand heavy
-				hitMode = HITMODE_TWFOHH;
+				hitMode = HITMODE_TWOHIT;
```

Comment:
Looks like we have logic changes here.  We had Two Weapon Fighting Off Hand Heavy {TWFOHH} changed to TWOHIT.